lib/peep.rb

```diff
@@ -0,0 +1,44 @@
+require 'pg'
+
+class Peep
+  attr_reader :id, :peep, :created_at
+
+  def initialize(id:, peep:, created_at:)
+    @id = id
+    @peep = peep
+    @created_at = created_at
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
```

Comment:
Something I have to do as well - it's good to dry up the data base connection based on enviroment variable into seperate file and wrap it into method there. It helps to make code look much cleaner especially later on where there is a lot of methods that connect to the data base.